openedx/core/djangoapps/credit/models.py

```diff
@@ -29,10 +29,47 @@ class CreditProvider(TimeStampedModel):
     get credit for course. Eligibility duration will be use to set duration
     for which credit eligible message appears on dashboard.
     """
+    provider_id = models.CharField(
+        max_length=255,
+        db_index=True,
```

Comment:
MySQL will create an index by default since this field is unique. There is no need for `db_index`.